ocaml/typing/primitive.ml

```diff
@@ -648,7 +648,7 @@ let prim_has_valid_reprs ~loc prim =
         is (Same_as_ocaml_repr C.word);
         any;
         is (Same_as_ocaml_repr C.value)]
-    | "%make_unboxed_tuple_vect" ->
```

Comment:
The thing that will be called `%makearray_dynamic`.

Reply:
Indeed, to quote an (easily missed) note in the PR description:

> The odd looking diff in typing/primitive.ml is a result of me accidentally including the typing rule for the new primitive prematurely in https://github.com/ocaml-flambda/flambda-backend/pull/3092, but with the name we planned when we thought it would only work with product arrays.